src/package/midlrt/Microsoft.Windows.MidlRT.targets

```diff
@@ -0,0 +1,339 @@
+    </Target>
+
+    <!--Insert Midl /references to Platform WinMDs, library reference WinMDs, and direct reference WinMDs-->
+    <Target Name="MidlRTSetMidlReferences"
+            Condition="'$(ModernMidlRT)' != 'false'"
+            DependsOnTargets="GetMidlRTPlatformWinMDReferences;GetMidlRTDirectWinMDReferences;GetMidlRTProjectWinMDReferences;$(MidlRTSetMidlReferencesDependsOn)"
+            Inputs="$(MSBuildAllProjects);@(MidlRTDirectWinMDReferences);@(MidlRTStaticProjectWinMDReferences);@(MidlRTDynamicProjectWinMDReferences);@(MidlRTPlatformWinMDReferences)"
+            Outputs="$(IntDir)midlrt.rsp">
+        <ItemGroup>
+            <_MidlReferences Remove="@(_MidlReferences)"/>
+            <_MidlReferences Include="@(MidlRTDirectWinMDReferences)"/>
+            <_MidlReferences Include="@(MidlRTStaticProjectWinMDReferences)"/>
+            <_MidlReferences Include="@(MidlRTDynamicProjectWinMDReferences)"/>
+            <_MidlReferences Include="@(MidlRTPlatformWinMDReferences)"/>
+            <_MidlReferencesDistinct Remove="@(_MidlReferencesDistinct)" />
+            <_MidlReferencesDistinct Include="@(_MidlReferences->'%(WinMDPath)'->Distinct())" />
+            <Midl Condition="'%(Midl.DisableReferences)'==''">
+                <AdditionalOptions>%(Midl.AdditionalOptions) %40"$(IntDir)midlrt.rsp"</AdditionalOptions>
+            </Midl>
+        </ItemGroup>
+        <PropertyGroup>
+            <_MidlrtParameters>@(_MidlReferencesDistinct->'/reference &quot;%(WinMDPath)&quot;','&#x0d;&#x0a;')</_MidlrtParameters>
+        </PropertyGroup>
+        <!-- Always write the midlrt.rsp file when the target runs, because the file is used as the output of this target. -->
+        <WriteLinesToFile
+            File="$(IntDir)midlrt.rsp" Lines="$(_MidlrtParameters)"
+            ContinueOnError="true" Overwrite="true" />
+        <Message Text="MidlRTMidlReferences: @(_MidlReferences->'%(WinMDPath)')" Importance="$(MidlRTVerbosity)"/>
+    </Target>
+
+    <!--Ctrl+F7 (selected file) midl compilation support-->
+    <Target Name="MidlRTSetSelectMidlReferences" BeforeTargets="SelectMidl" DependsOnTargets="MidlRTSetMidlReferences" />
+
+    <!--Merge project-generated WinMDs and project-referenced static library WinMDs into project WinMD-->
+    <Target Name="MidlRTMergeProjectWinMDInputs"
+            DependsOnTargets="Midl;GetMidlRTMdMergeInputs;$(MidlRTMergeProjectWinMDInputsDependsOn)"
+            Inputs="$(MSBuildAllProjects);@(MidlRTMdMergeInputs)"
+            Outputs="@(_MdMergedOutput);$(IntDir)mdmerge.rsp">
+        <PropertyGroup>
+            <!--Note: MidlRTNamespaceMergeDepth supersedes MidlRTMergeDepth-->
+            <_MdMergeDepth Condition="'$(MidlRTNamespaceMergeDepth)' != ''">-n:$(MidlRTNamespaceMergeDepth)</_MdMergeDepth>
+            <_MdMergeDepth Condition="'$(_MdMergeDepth)' == ''">$(MidlRTMergeDepth)</_MdMergeDepth>
```

Comment:
since this is new, don't need to carry the c++/winrt baggage (delete MidlRTMergeDepth)

Reply:
Gotcha, sounds good!